services/data-preparation.service.ts

```diff
@@ -1046,4 +1102,61 @@ export const dataPreparationService = {
     );
     return response.data;
   },
+
+  async retrieveCurrentRiskAnalysisVersion(token: string) {
+    const response =
+      await apiClient.purposes.retrieveLatestRiskAnalysisConfiguration(
+        getAuthorizationHeader(token)
+      );
+
+    assertValidResponse(response);
+
+    return response.data.version;
```

Comment:
```suggestion
    return response.data;
```
For reuse I would return the whole object so we can get what we need whenever we call it